Rework Installation into Distribution Channels (#131)
DevExpress/devextreme-documentation: concepts/Common/Distribution Channels/20 Installer for Windows.md

The installer for Windows provides Visual Studio integration features, such as project scaffolding. Use it if you develop applications in Visual Studio.

> After the installation, you will be able to find DevExtreme sources in the %ProgramFiles(x86)%\DevExpress 20.1\DevExtreme folder.

Reviewer: ```suggestion
After the installation, you can find the DevExtreme sources in the %ProgramFiles(x86)%\DevExpress 20.1\DevExtreme folder.
```
